class/defaults.yml

```diff
@@ -1,3 +1,11 @@
 parameters:
   metallb:
-    namespace: syn-metallb
+    namespace: metallb-system
+    secretkey: none 
+    # ?{vaultkv:${customer:name}/${cluster:name}/metallb/secretkey}
+    # kubectl create secret generic -n metallb-system memberlist --from-literal=secretkey="$(openssl rand -base64 128)"
```

Comment:
What is the purpose of that `kubectl` command? If it is important for using this component, this has to go into documentation. Otherwise, it has to be omitted.

```suggestion
```

Reply:
It's now a secret reference to Vault